### PR TITLE
Prepare v0.31.0 release

### PR DESCRIPTION
**Pull request by srenatus**
🎉 Please review the release notes 📜

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,192 @@
...
+- New configuration for Management APIs: using `resource`, the request path for sending decision logs can be configured now ([#3618](https://github.com/open-policy-agent/opa/issues/3618)), authored by @[cbuto](https://github.com/cbuto)
+
+  `/logs` is still the default, but can now be overridden.
+  With this change, the `partition_name` config becomes deprecated, since its functionality is subsumed by this new configurable.
+
+### Documentation
+
+- How to debug? Clarify how to access `Note` events for debugging via explanations ([#3628](https://github.com/open-policy-agent/opa/issues/3628)) authored by @[enori](https://github.com/enori)
+- Clarify special characters for key, i.e. what `x["y"]` is necessary because `x.y` isn't valid ([#3638](https://github.com/open-policy-agent/opa/issues/3638)) authored by @[Hongbo-Miao](https://github.com/Hongbo-Miao)
+- Integrations: add Python Bottle library ([#3619](https://github.com/open-policy-agent/opa/pull/3619)), by @[dolevf](https://github.com/dolevf)
```
Comment (anderseknert) [inline CHANGELOG.md:189]: While I don't mind this, I don't think we've listed added integrations in the past, as they aren't really attached to a release, and are immediately seen on the ecosystem page.

**Review by anderseknert — APPROVED** [review]
Looks good!

File: CHANGELOG.md (hunk 1 above)
Comment (srenatus) [reply CHANGELOG.md:189]: Oops I wasn't aware that they turned up immediately. 💡 I'll remove this, thanks.

**Review by anderseknert — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,191 @@
...
+
+### Tooling
+
+- Query Profiling: `opa eval --profile` now supports a `--count=#` flag to gather metrics and profiling data over multiple runs, and displays aggregate statistics for the results ([#3651](https://github.com/open-policy-agent/opa/issues/3651)).
+
+  This allows you to gather more robust numbers to assess policy performance.
+
+- Docker images: Publish static image ([#3633](https://github.com/open-policy-agent/opa/issues/3633))
+
+  As of this release, you can use the staticly-built Linux binary from a docker image: `openpolicyagent/opa:v0.31.0-static`.
```
Comment (tsandall) [inline CHANGELOG.md:165]: Remove the `v` from the image tag. This is just a legacy thing; the makefile doesn't include "v" in the image tags.

**Review by tsandall — COMMENTED** [review]
Let's get the console logger fix into the release notes and then this LGTM.

**Review by ashutosh-narkar — APPROVED** [review]
LGTM

**Pull request merged**